### DISCUSSION TITLE
Upload proof of academic status "take a picture" button nonfunctional

### DISCUSSION THREAD

**Discussion by Tomaster134** [community/community#88616] [GitHub Education]
### Hi everyone,

I'm attempting to verify my status as a student, but the "take a picture" button does nothing when clicked. When I attempt to cancel the application I get a 404 error, and if I try to process the application I get 
```
Your application cannot be reviewed until you fix the following:

    Your request has expired, please try again
```

I'm not sure how to proceed here.

**Comment by gina-v** [comment]
I'm having the same issue. I'm stuck on the 'Upload Proof of Your Academic Status' page. The 'take a picture' button is not working. If I try to upload a document and submit the application I get the same message. If I try canceling the application I get a 404 error.
